src/components/elementView/elementTab.tsx

```diff
@@ -425,15 +435,9 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
                 disabled={
                   this.props.changePlan
                     ? true
```

Comment:
This means if you're in a change plan, you can't bulk upload anything - that's desired, right?

Reply:
yep